modules/image-loading-optimization/module.php

```diff
@@ -0,0 +1,346 @@
+<?php
+namespace Elementor\Modules\ImageLoadingOptimization;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+	/**
+	 * @var string The experiment name.
+	 */
+	const EXPERIMENT_NAME = 'e_image_loading_optimization';
+
+	/**
+	 * @var int Minimum square-pixels threshold.
+	 */
+	private $min_priority_img_pixels = 50000;
```

Comment:
Should there be a filter for this value (as there is in WP)?

Reply:
Yes, we should add one here. Good catch